spec/content/manifest.spec.yml

```diff
@@ -0,0 +1,70 @@
+##
+## Describes the specification for the content package's main manifest.yml file
+##
+spec:
+  # Everything under here follows JSON schema (https://json-schema.org/), written as YAML for readability
+  type: object
+  additionalProperties: false
+  definitions:
+    interested_on:
```

Comment:
This seems like a tricky thing to name. What do you think about `discovery_fields` instead, in the interest of being more explicit about the behavior associated with this property?

Reply:
Yeah, I am not super happy with the name, so thanks for the suggestion!

I wouldn't include `fields` at this level because maybe we can support other things, like other packages, but `discovery` could fit.
```
discovery:
  fields:
    - name: "process.pid"
```
```
discovery:
  packages:
    - name: "system"
```

Reply:
I like leaving room for driving recommendations for other packages vs fields - I think this approach is solid.